assets/sass/local.scss

```diff
@@ -293,3 +293,8 @@
     @extend .govuk-tag--orange;
   }
 }
+
+// fix colour on MoJ sortable table for non-sortable headings
```

Comment:
Used for the 'Action' column which we don't want to be sortable. Without this it defaults to black rather than blue like the sortable headers.